doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -156,9 +156,15 @@ nRF9160: Asset Tracker v2
 nRF9160: Serial LTE modem
 -------------------------
 
-* Added an RFC1350 TFTP client, currently supporting only *READ REQUEST*.
-* Added new AT command #XSHUTDOWN to put nRF9160 SiP to System OFF mode.
-* Added support to nRF Cloud C2D appId "MODEM" and "DEVICE".
+* Added:
+
+  * Added an RFC1350 TFTP client, currently supporting only *READ REQUEST*.
+  * Added new AT command #XSHUTDOWN to put nRF9160 SiP to System OFF mode.
+  * Added support to nRF Cloud C2D appId "MODEM" and "DEVICE".
```

Comment:
```suggestion
  * An RFC1350 TFTP client, currently supporting only *READ REQUEST*.
  * New AT command #XSHUTDOWN to put nRF9160 SiP to System OFF mode.
  * Support to nRF Cloud C2D appId "MODEM" and "DEVICE".
```
No need to repeat the "Added" since it's already the lead-in for these bullets.